src/mam4xx/lin_strat_chem.hpp

```diff
@@ -0,0 +1,314 @@
+  // used only if abs(latitude) > lats_threshold
+  psc_activation(lats, temperature, pmid, sza, linoz_cariolle_psc,
+                 delta_t,                                         // in
+                 excess_chlorine, o3_old,                         // in
+                 psc_T, chlorine_loading, o3_new, do3_linoz_psc); // out
+
+  o3_vmr = o3_new; // update ozone vmr
+
+  sza_degrees = sza * radians_to_degrees;
+
+} // lin_strat_chem_solve_kk
+
+KOKKOS_INLINE_FUNCTION
+void lin_strat_chem_solve(
+    const ThreadTeam &team, const ColumnView &o3col,
+    const ColumnView &temperature, const Real sza, const ColumnView &pmid,
+    const Real delta_t, const Real rlats,
+    // ltrop, & !in
+    const ColumnView &linoz_o3_clim, const ColumnView &linoz_t_clim,
+    const ColumnView &linoz_o3col_clim, const ColumnView &linoz_PmL_clim,
+    const ColumnView &linoz_dPmL_dO3,
+    const ColumnView &linoz_dPmL_dT, // in
+    const ColumnView &linoz_dPmL_dO3col,
+    const ColumnView &linoz_cariolle_psc, // in
+    const int ltrop, const Real chlorine_loading,
+    const Real psc_T, // PSC ozone loss T (K) threshold
+    const ColumnView &o3_vmr,
+    // diagnostic variables outputs
+    const ColumnView &do3_linoz, const ColumnView &do3_linoz_psc,
+    const ColumnView &ss_o3, const ColumnView &o3col_du_diag,
+    const ColumnView &o3clim_linoz_diag, const ColumnView &sza_degrees) {
+
+  Kokkos::parallel_for(
+      Kokkos::TeamThreadRange(team, ltrop), KOKKOS_LAMBDA(int kk) {
+        lin_strat_chem_solve_kk(
+            o3col(kk), temperature(kk), sza, pmid(kk), delta_t, rlats,
+            // ltrop, & !in
+            linoz_o3_clim(kk), linoz_t_clim(kk), linoz_o3col_clim(kk),
+            linoz_PmL_clim(kk), linoz_dPmL_dO3(kk),
+            linoz_dPmL_dT(kk), // in
+            linoz_dPmL_dO3col(kk),
+            linoz_cariolle_psc(kk), // in
+            chlorine_loading,
+            psc_T, // PSC ozone loss T (K) threshold
+            o3_vmr(kk),
+            // diagnostic variables outputs
+            do3_linoz(kk), do3_linoz_psc(kk), ss_o3(kk), o3col_du_diag(kk),
+            o3clim_linoz_diag(kk), sza_degrees(kk));
+      });
+
+} // lin_strat_chem_solve
+
+KOKKOS_INLINE_FUNCTION
+void lin_strat_sfcsink(const Real delta_t, const ColumnView &pdel, // in
+                       const ColumnView &o3l_vmr, const Real o3_sfc,
+                       const int o3_lbl, const Real o3_tau, Real &o3l_sfcsink) {
+
+  // @param[in] delta_t              timestep size [secs]
+  // @param[in] pdel(ncol,pver)      pressure delta about midpoints [Pa]
+
+  // !inten in-outs
+  // @param[inout]:: o3l_vmr(ncol ,pver)             ! ozone volume mixing ratio
+  // [vmr]
+
+  constexpr Real one = 1.0;
+  // BAD CONSTANT
+  constexpr Real mwo3 = 48.; // molecular weight O3
+  constexpr Real mwdry = haero::Constants::molec_weight_dry_air *
+                         1e3; //     ! molecular weight dry air ~ kg/kmole;//!
```

Comment:
So 'molec_weight_dry_air' must have units of g / kmole?

Reply:
`molec_weight_dry_air` has units of kg/mol; see[ here](https://github.com/eagles-project/haero/blob/d7d3e548a4271d9c9b7026c7e189425967a9a78e/haero/constants.hpp#L59).
Thus, we need to multiple by a factor of 1e3, i.e.,`kg/mol*(1e3/k) =>  kg/kmol`